src/backend/core/models.py

```diff
@@ -452,6 +452,27 @@ def get_abilities(self, user):
         }
 
 
+class Invitation(BaseModel):
```

Comment:
NIceeee, I would suggest adding
- A foreign key to the sender/issuer of the invitation.
- A validity duration, with a default value of 7 days? Or an expiration date

So we match Github invitation: 
```
@[mjeammet](https://github.com/mjeammet) has invited you to join the @numerique-gouv organization on GitHub. Head over to https://github.com/numerique-gouv to check out @numerique-gouv’s profile.

This invitation will expire in 7 days.
```

Reply:
Hey, cool proposals ! About the sender/issuer, its pretty straightforward. For the validity duration, should I actually transform `is_pending` to a `status` field, with three choices : "pending", "expired", "archived/resolved" ?

Reply:
I am pretty sure we can challenge the `is_pending` boolean field. As now, `is_pending` can not be falsy. If a user accepts or rejects the invitation, we would delete the row, meaning that only rows with ìs_pending === true` would be stored in this table. IMO, this attribute as boolean doesn't provide much value, but I might be wrong, and forget some cases.

Reply:
OK ! So, as discuted on tchap : 

- `is_pending` is removed. Invitation status will be calculated via a property 
- expired invitations are kept in database, to allow for re-invitations

Reply:
In this model, accepted invitations are deleted and converted to team accesses.